ci-operator/config/securesign/cosign/OWNERS

```diff
@@ -0,0 +1,7 @@
+# The OWNERS file is used by prow to automatically merge approved PRs.
+
+approvers:
+- lance
+- sabre1041
+- cooktheryan
+- sallyom
```

Comment:
@JasonPowr if you add yourself and @tommyd450 to this file, you both will have /lgtm and /approve rights on this repo.